graspy/utils/utils.py

```diff
@@ -217,42 +218,64 @@ def remove_loops(graph):
     return graph
 
 
-def to_laplace(graph, form="DAD"):
+def to_laplace(graph, form="DAD", regularizer=None):
     r"""
     A function to convert graph adjacency matrix to graph laplacian. 
 
-    Currently supports I-DAD and DAD laplacians, where D is the diagonal
+    Currently supports I-DAD, DAD, and R-DAD laplacians, where D is the diagonal
     matrix of degrees of each node raised to the -1/2 power, I is the 
-    identity matrix, and A is the adjacency matrix
+    identity matrix, and A is the adjacency matrix.
+    
+    R-DAD is regularized laplacian: where :math:`D_t = D + regularizer*I`
+    regularizer defaults to the average node degree.
 
     Parameters
     ----------
     graph: object
         Either array-like, (n_vertices, n_vertices) numpy array,
         or an object of type networkx.Graph.
 
-    form: {'I-DAD' (default), 'DAD'}, string, optional
+    form: {'I-DAD' (default), 'DAD', 'R-DAD'}, string, optional
         
         - 'I-DAD'
             Computes :math:`L = I - D*A*D`
         - 'DAD'
             Computes :math:`L = D*A*D`
+        - 'R-DAD'
+            Computes :math:`L = D_t*A*D_t` where :math:`D_t = D + regularizer*I`
+    regularizer: scalar
```

Comment:
specify ignored it form is not r-dad

Reply:
This should specify datatype. E.g. int or float and that it should be >= 0.